packages/twenty-front/src/modules/object-record/record-table/hooks/useComputeNewRowPosition.ts

```diff
@@ -1,90 +0,0 @@
-import { DropResult } from '@hello-pangea/dnd';
```

Comment:
removing this hook as we should leave recordBoard, recordTable, recordTableWithGroups implement their way of selecting the right records + we already have getDraggedRecordPosition helper